week1/ditto.html

```diff
@@ -8,15 +8,23 @@
     <script>
```

Comment:
This file shouldn't be part of this PR in the future. Each PR should only have the homework assignment you are working on.